src/Sylius/Behat/Context/Setup/ChannelContext.php

```diff
@@ -127,6 +128,26 @@ public function theChannelIsDisabled(ChannelInterface $channel)
     }
 
     /**
+     * @Given /^(this channel) has (currency "([^"]+)")$/
+     */
+    public function thisChannelHasCurrency(ChannelInterface $channel, CurrencyInterface $currency)
+    {
+        $channel->addCurrency($currency);
+        $this->channelManager->flush();
+        $this->sharedStorage->set('channel', $channel);
+    }
+
+    /**
+     * @Given /^(this channel) host is "([^"]+)"$/
+     */
+    public function thisChannelHostIs(ChannelInterface $channel, $hostName)
```

Comment:
Why we need this? See https://github.com/Sylius/Sylius/blob/master/features/channel/theming/displaying_themed_channel_website.feature and its implementation in Behat's contexts to see how to switch between channels in test/dev env. :)